.murdock

```diff
@@ -168,10 +174,14 @@ get_app_board_toolchain_pairs() {
 
 # use dwqc to create full "appdir board toolchain" compile job list
 get_compile_jobs() {
-    get_apps | \
-        dwqc ${DWQ_ENV} -s \
-        ${DWQ_JOBID:+--subjob} \
-        "$0 get_app_board_toolchain_pairs \${1} $0 compile"
+    if [ $DISABLE_COMPILE_TESTS != 1 ]; then
+        get_apps | \
+            dwqc ${DWQ_ENV} -s \
+            ${DWQ_JOBID:+--subjob} \
+            "$0 get_app_board_toolchain_pairs \${1} $0 compile"
+    else
+        return 1
```

Comment:
Not sure this is right or required. I basically copied the behavior from `test_cache_get()`

Reply:
`get_non_compile_jobs()` lets me guess on the otherhand the function can just return nothing, so I do this here as well (amended the change directly).